src/advanced-calculator/index.tsx

```diff
@@ -435,7 +435,7 @@ export default function Calculator() {
             </ul>
           </InView>
           <InView>
-            <h5>Familie</h5>
+            <h4 className="var-heading var-heading--5">Familie</h4>
```

Comment:
Feel like this at some point could be it's own title + level + override element component that can be reused on different systems. Now we've been mixing style and semantics

Reply:
Created a Heading component for reuse